test/fixtures/users.yml

```diff
@@ -0,0 +1,11 @@
+# Read about fixtures at https://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
```

Comment:
pls remove this file and all auto-generated "tests", we like to write our tests))